Pod/Classes/XExtensionItem.h

```diff
@@ -89,6 +126,13 @@ typedef UIImage *(^XExtensionItemThumbnailProvidingBlock)(CGSize suggestedSize,
 @property (nonatomic) XExtensionItemReferrer *referrer;
 
 /**
+ A block that will be called with the suggested size for the thumbnail image, in points, and the activity type chosen by 
+ the user. It should return an image using the appropriate scale for the screen. Images provided at the suggested size 
+ will result in the best experience.
+ */
+@property (nonatomic, copy) XExtensionItemThumbnailProvidingBlock thumbnailProvider;
```

Comment:
That certainly would be better, but we went with this in order to match `XExtensionItemProvidingBlock`, which is so named to avoid calling it `XExtensionItemProviderBlock` which we worried could be confusing since that makes it sound like the block returns a `NSItemProvider`, but it doesn't. Happy to hear any other proposals.